test/integration/build_admission_test.go

```diff
@@ -13,165 +13,148 @@ import (
 	"github.com/openshift/origin/pkg/client"
 	policy "github.com/openshift/origin/pkg/cmd/admin/policy"
 	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
+	imageapi "github.com/openshift/origin/pkg/image/api"
 	testutil "github.com/openshift/origin/test/util"
 )
 
 func TestPolicyBasedRestrictionOfBuildStrategies(t *testing.T) {
```

Comment:
maybe combine the create/clone tests to save setup/test time:
1. verify create succeeds and save result
2. verify clone of saved succeeds
3. remove perms
4. verify create fails
5. verify clone of saved build fails